sosivio/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d98f7912-e5a3-48bc-b63e-612d835bf6b4",
+  "app_id": "sosivio",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Get Answers. Not Data. Predictive Troubleshooting for Kubernetes.",
+    "title": "Sosivio",
+    "media": [],
```

Comment:
@DanArlowski please add at least one image to your tile - for example, it can be a screenshot of the Datadog dashboard or the Sosivio UI. To add an image, use the following syntax:

```
"media_type": "image",
"caption": "caption goes here",
"image_url": "relative-path-to-image.png"
```

Reply:
@anawishnoff , I've added one image - regardless - once we'll do a public `true` we'll add more :)

Reply:
@anawishnoff  - not sure how i missed the image,
added it now

and im aware.
thanks!